### PR TITLE
Adding Presentation Photos

### PR DESCRIPTION
**Pull request by devinmatte**
Simply adds photos to the presentations
image

### CODE REVIEW

File: conditional/templates/intro_eval_slideshow.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
     {% for m in members %}
     <section id="slide-{{m['uid']}}">
       <section>
+        <img class="eval-user-img" alt="{{m['uid']}}" src="https://profiles.csh.rit.edu/image/{{m['uid']}}" width="100" height="100" />
```
Comment (RamZallan) [inline conditional/templates/intro_eval_slideshow.html]: Any reason you're doing the styles inline here (and also in `spring_eval_slideshow.html`) if you're already changing some CSS?

Comment (devinmatte) [reply conditional/templates/intro_eval_slideshow.html]: Oops. Left that in by accident

Comment (devinmatte) [reply conditional/templates/intro_eval_slideshow.html]: What do you mean? I don't see any inline css here? Only the width and height setting

Comment (RamZallan) [reply conditional/templates/intro_eval_slideshow.html]: Right, that's what I'm referencing. I guess my wording was off, but it's still "styling" them. Just wondering why you didn't just set
```
    width: 100px;
    height: 100px;
```
instead. Guess this is more of a nitpick, though, that's all I found to comment on.

**Comment by mbillow** [comment]
:shipit:

**Pull request merged**